docs/source/notes/cuda.rst

```diff
@@ -132,59 +147,88 @@ flag that can be used to cast tensors and modules to CUDA if desired::
     parser.add_argument('--disable-cuda', action='store_true',
                         help='Disable CUDA')
     args = parser.parse_args()
-    args.cuda = not args.disable_cuda and torch.cuda.is_available()
+    args.device = None
```

Comment:
Why assign `None` to it when you could have just made it `torch.device('cpu')` by default.